Backend/ManagerLayer/UserManagement/UserManagementManager.cs

```diff
@@ -59,80 +61,44 @@ public User CreateUser(DatabaseContext _db, string email, Guid SSOID)
             return user;
         }
 
-        public int DeleteUser(User user)
+        public void DeleteUser(Guid id)
         {
-            using (var _db = CreateDbContext())
-            {
-                var response = _userService.DeleteUser(_db, user.Id);
-                // will return null if user does not exist
-                return _db.SaveChanges();
-            }
-        }
-
-        public int DeleteUser(Guid id)
-        {
-            using (var _db = CreateDbContext())
-            {
-                var response = _userService.DeleteUser(_db, id);
-                return _db.SaveChanges();
-            }
+            _userService.DeleteUser(_db, id);
         }
 
         public User GetUser(Guid id)
         {
-            using (var _db = CreateDbContext())
-            {
-                return _userService.GetUser(_db, id);
-            }
+            return _userService.GetUser(_db, id); 
         }
 
         public User GetUser(string email)
         {
-            using (var _db = CreateDbContext())
-            {
-                return  _userService.GetUser(_db, email);
-            }
+            return _userService.GetUser(_db, email);
         }
 
-        public int DisableUser(User user)
+        public void DisableUser(User user)
         {
-            return ToggleUser(user, false);
+            ToggleUser(user, true);
         }
 
-        public int EnableUser(User user)
+        public void EnableUser(User user)
         {
-            return ToggleUser(user, true);
+            ToggleUser(user, false);
         }
 
-        public int ToggleUser(User user, bool? enable)
+        public void ToggleUser(User user, bool? disable)
         {
-            using (var _db = CreateDbContext())
+            if (disable == null)

```

Comment:
I didn't write the code but i think its meant for is you want to call toggle and just pass in the user rather than determining to toggle them as true or false